test/metabase/api/timeline_test.clj

```diff
@@ -127,7 +178,7 @@
                                                 :timeline_id (u/the-id timeline)
                                                 :archived    true}]]
           (testing "a timeline with no events returns an empty list"
-            (is (= #{} (event-names (include-events-request empty-tl false)))))
+            (is (= '() (:events (include-events-request empty-tl false)))))
```

Comment:
no need to change now but you don't need to quote the empty list in Clojure. And equality will check across collection type so you could use a more common `[]`. or just assert that it is `empty?`